src/app/search-page/searchPage.controller.js

```diff
@@ -54,16 +54,7 @@ export default class SearchPageController {
                 escapeToClose: true
             })
             .then((answer) => {
-                //Implement here email sending service, bottom toast on mail sending
-                //add another toast for email error sending
-                console.log(answer);
-                this.$mdToast.show(
-                    this.$mdToast.simple()
-                    .textContent('We will contact you soon!')
-                    .position('bottom right')
-                    .theme('success-toast')
-                    .hideDelay(3000)
-                );
+                this.mailingService.contact(answer);
             }).catch(() => {
```

Comment:
If you are not doing anything here then you don't have to put the catch, if you put the catch then you have to do something

Reply:
Hm, I did this because if I didn't put a catch it throws an error of "possibly unhandled exception"